background/services/provider-factory/provider-factory.ts

```diff
@@ -109,12 +105,6 @@ export default class ProviderFactory extends BaseService<ProviderFactoryEvents>
   }
 
   private emitLocalNodeNetworkStatusEvent(status: boolean): void {
-    if (
```

Comment:
after this.isLocalNodeNetworkProvidersInitialized = true inside initializeProvidersForLocalNodeNetwork()

we need to reset this.lastLocalNodeStatus properly